crates/astria-proto/buf.gen.yaml

```diff
@@ -6,5 +6,6 @@ plugins:
     out: .
     opt:
      - extern_path=.tendermint.types=::tendermint-proto::types
+     - type_attribute=.astria.primitive.v1.Uint128=#[derive(Copy)]
```

Comment:
Kinda nice.

I wonder how maintainable this will be in the long run should we ever want to add more type attributes or other things. But it's what we are given. :sweat_smile: : :shrug: